dist/obs-server.spec

```diff
@@ -484,7 +484,7 @@ for i in production.rb ; do
 done
 SECRET_KEY="/srv/www/obs/api/config/secret.key"
 if [ ! -e "$SECRET_KEY" ]; then
-  ( umask 0077; dd if=/dev/urandom bs=256 count=1 2>/dev/null |sha256sum| cut -d\  -f 1 >$SECRET_KEY )
+  RAILS_ENV=production bundle exec rake secret > $SECRET_KEY
```

Comment:
It's just using rake. The footprint seems to be quite small, 200KiB memory and around 20% cpu load for a few seconds...
And this way we won't have to maintain the creation of generating the secret.

Regarding the supposed security hole, I'll have to check the source code. But then this would be a problem for other rails apps